src/main/python/__cvd_risk_bmi.py

```diff
@@ -86,22 +91,44 @@ def heart_age_by_cvd_bmi(cvd,age,sex):
     #handle the condition wheere cvd is None
     if cvd is None:
         return 0
-    if str(cvd) == n_a:
+    if str(cvd) == "":
         return 0
     else:
         cvd = float(cvd)
     if (sex == 'Male'):
-        X = BETA_MEN[1]*np.log(22) - CONST_MEN
+        X = BETA_MEN[1]*np.log(22) - CONST_MEN + CONST_MEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_MEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_MEN[0])))
         return np.round(result)
-    else
-        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN
+    else:
+        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN + CONST_WOMEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_WOMEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_WOMEN[0])))
         return np.round(result)
 
+def validate_heart_age(cvd,age,sex):
+    #handle the condition wheere cvd is None
+    if cvd is None:
+        return 0
+    if str(cvd) == "":
+        return 0
+    else:
+       split_gender = sex.split(",")
+       return 1
+
+def add_heart_age_adjuster(heart_age):
+    heart_age_adjuster : int = 1
+    
+    print("Using heart age adjuster..")
+
+    print("test...")
+    return heart_age + heart_age_adjuster
+
 #cvd = frs('Male',10,33,26.3,0,0,1)
-#print heart_age_by_cvd(cvd,32,'Male')
+print(heart_age_by_cvd_bmi(cvd,32,'Male'))
+heart_age = heart_age_by_cvd_bmi(cvd,32,'Female')
+hear_age_post_adjustment = add_heart_age_adjuster(heart_age)
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/adhirpotdarbito/pranahealth/pull/6#issuecomment-2304109346>#e78355</a><span> - 08/22/2024, 08:47 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The heart_age variable is used without checking if it's a valid value before passing it to add_heart_age_adjuster.
</div>
<div id="fix">
<b>Fix:</b> Add a check to ensure heart_age is a valid value before using it in add_heart_age_adjuster.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -131,2 +131,5 @@
  heart_age = heart_age_by_cvd_bmi(cvd,32,'Female')
 -hear_age_post_adjustment = add_heart_age_adjuster(heart_age)
 +if isinstance(heart_age, (int, float)) and heart_age >= 0:
 +    heart_age_post_adjustment = add_heart_age_adjuster(heart_age)
 +else:
 +    print("Invalid heart age value")
 +    heart_age_post_adjustment = None
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged